hack/addons/kustomize/nutanix-snapshot-csi/kustomization.yaml.tmpl

```diff
@@ -16,4 +16,3 @@ helmCharts:
   version: ${NUTANIX_SNAPSHOT_CSI_CHART_VERSION}
   includeCRDs: true
   skipTests: true
-  namespace: nutanix-system
```

Comment:
What is this change for?

Reply:
It's not used (these files aren't actually used at all other than for generating the helm chart versions configmap used internally). It's actually deployed into `ntnx-system` but i didn't want to leave this here as it is confusing to me.